utils/Command/MissingInSetCommand.php

```diff
@@ -141,7 +141,9 @@ static function (string $rectorClass): bool {
             $hasError = true;
             $this->symfonyStyle->title('We could not find there rules in configs');
 
-            $setRealpath = (string) realpath($setFile);
+            /** @var string|false $setRealpath */
+            $setRealpath = realpath($setFile);
+            $setRealpath = (string) $setRealpath;
             $relativeFilePath = Strings::after($setRealpath, getcwd() . '/');
```

Comment:
This will be always `string`, as we use existing set paths.

Reply:
sure, I updated at :

- https://github.com/rectorphp/rector-src/pull/5040

Reply:
Thanks :clap:

Reply:
@staabm that's what I thought, but since `tools/` directory is not shipped to `rector/rector`, I guess it safe for user, the concern is probably when directory/set removed in the future and not detected.

The better solution is probably use:

```php
Assert::string($setRealpath);
```

so it will be detected early when path removed/renamed.